.github/ISSUE_TEMPLATE/patternfly-issue-template.md

```diff
@@ -0,0 +1,11 @@
+Please populate the PatternFly issue template so we can learn more about your needs.
```

Comment:
This line needs to be commented. Also, please add this codesandbox link for people to easily reproduce issues: https://codesandbox.io/s/x2isc

Reply:
What do you mean by it needs to be commented? It is just a general description of the template. I can add the link for sure.

Reply:
I mean it needs to look like this: ```
<!-- Please populate the PatternFly issue template so we can learn more about your needs. Here's a codesandbox link you can reproduce your issue in: https://codesandbox.io/s/x2isc -->
```

The `<!-- -->` makes it a comment that won't appear as plain text inside the issue.